sdks/golang/docs/ListPipelinesResponse.md

```
@@ -0,0 +1,72 @@
+# ListPipelinesResponse
+
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**Pipelines** | [**[]PipelineView**](PipelineView.md) |  | 
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)